pkg/services/oauthtoken/oauth_token_test.go

```diff
@@ -264,3 +286,202 @@ func (f *FakeAuthInfoStore) UpdateAuthInfo(ctx context.Context, cmd *login.Updat
 func (f *FakeAuthInfoStore) DeleteAuthInfo(ctx context.Context, cmd *login.DeleteAuthInfoCommand) error {
 	return f.ExpectedError
 }
+
+func TestService_TryTokenRefresh(t *testing.T) {
+	type testCase struct {
+		desc      string
+		identity  *authn.Identity
+		oauthInfo *social.OAuthInfo
+		setupEnv  func(cache *localcache.CacheService, authInfoService *authinfotest.FakeService)
+
+		expectHasEntryCalled bool
```

Comment:
Looks like this is a miss, we don't check it in the test.

Reply:
I removed the property as it is not being used anywhere